templates/repo/wiki/new.tmpl

```diff
@@ -16,6 +16,14 @@
 			<div class="field {{if .Err_Title}}error{{end}}">
 				<input name="title" value="{{.title}}" autofocus required>
 			</div>
+			<div class="help">
+				{{.i18n.Tr "repo.wiki.page_name_desc"}}
+				<ul>
+					<li>{{.i18n.Tr "repo.wiki.page_name_desc_home"}}</li>
+					<li>{{.i18n.Tr "repo.wiki.page_name_desc_sidebar"}}</li>
+					<li>{{.i18n.Tr "repo.wiki.page_name_desc_footer"}}</li>
+				</ul>
+			</div>
```

Comment:
These lines occupy too much space (and usually users are not interested at them).

Maybe one line help (or tool tip) is enough: `Special Wiki page names: Home / _Sidebar / _Footer`

Reply:
Makes sense. Fixed in 3a3bbc8.

Reply:
The screenshot in the first comment can be updated also.